addons/binding/org.openhab.binding.meterreader/.project

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.openhab.binding.meterreader</name>
```

Comment:
"MeterReader" is imho a bit too generic - there exist many different meters in the world and we e.g. already have the DSMR binding for the netherlands.
My suggestion would be to simply go for "SML Binding" and use "sml" as the binding id. Wdyt?

Reply:
Agree that meterreader is too generic. But the binding is also supporting IEC62056-21, which is why I renamed the binding.
IEC62056-21 is supported already by @kreutpet binding (which is not yet ported to openhab2).  Would you suggest to remove this feature (few people are already using it)?

Reply:
Hm, can you explain my how SML and IEC62056-21 related to each other? I thought to understand that SML is the data exchange specification, while IEC62056-21 defines a transport layer. So what would it mean to "remove this feature"? If meters that use IEC62056-21 do NOT also use SML at the same time, I would indeed suggest to implement a separate binding for them assuming that not much code is in common with the SML-ones anyhow.

Reply:
SML is an exchange format. The meter can simply PUSH SML messages at any time (The client can also send data which we don't need here).
IEC62056-21 defines a data format and a transport protocol. It has different modes (A,B,C,D and E) which basically are differentiated by PUSH or PULL mode and how to exchange communication parameters.
The things that are in common:
- Data is identified by OBIS codes
- Data has a value and a unit

-> That is all we need for the binding to work basically. Hence also the infrastructure code is in common. Only difference is in how we read the data from the meter (and that is mostly abstracted by the respective library).

But the question stays. Do we want to support different standards in one binding? We could e.g. also add DLMS/COSEM support to that binding...

It has clearly some advantages (DRY principle, easier to maintain as many code is in common and 
needs only be explained once, ...) but also disadvantage (The "main" maintainer usually can only test one of the standards, more difficult architecture, more code (can be more complicated)).

Reply:
I changed the binding's name to "smartmeter"